docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -122,6 +122,8 @@ Preparing For a Release
       mkvirtualenv flocker-release
       pip install --editable .[dev]
       admin/create-release-branch --flocker-version="${VERSION}"
+      admin/update-license
+      git commit -am "Updated copyright in LICENSE file"
```

Comment:
:question: Comment that this last line exits with 1 if the license does not need to be updated. May be useful to remember if this step is automated in the future.